engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -221,6 +227,10 @@ namespace dmPlatform
         }
 
         glfwWindowHint(GLFW_FOCUSED, GLFW_TRUE);
+        // This hint saves both size and position, but we need only position
+        // That's why we hide window here and then later set default size and show window
+        glfwWindowHintString(GLFW_COCOA_FRAME_NAME, params.m_Title);
+        glfwWindowHint(GLFW_VISIBLE, GLFW_FALSE);
```

Comment:
This hint turn saves for the window frame (position and size). But we need only position save. 
So we don't auto-show window on creation using `GLFW_VISIBLE` and then show later after we change size back

Reply:
This whole workaround is needed to prevent behavior when each restart moves window a little bit bottom and right:

https://github.com/defold/defold/assets/2209596/317466ca-f340-40e9-84c2-b959a1bb820a

As a side effect we get window position save function on mac. This feature were requested a few times.